messages/ca.json

```diff
@@ -0,0 +1,27 @@
+  
+{
+  "store/product-list.pricePerUnit": "{price} {perMeasurementUnit}",
+  "store/product-list.pricePerUnit.measurementUnit": "per {unitMultiplier} {measurementUnit}.",
+  "store/product-list.availableItems": "{quantity, plural, one {# producte disponible} other {# productes disponibles}}",
+  "store/product-list.unavailableItems": "{quantity, plural, one {# producte no disponible} other {# productes no disponibles}}",
+  "store/product-list.message.cantBeDelivered": "Aquest producte no es pot lliurar a l'adreça proporcionada.",
```

Comment:
"Aquest producte no es pot lliurar a l'adreça facilitada."

Reply:
hi @camilavcoutinho we created a new PR to fix all these comments:
https://github.com/vtex-apps/product-list/pull/96